src/component/legend/LegendView.ts

```diff
@@ -564,8 +568,11 @@ function getLegendStyle(
     const legendItemModel = legendModel.getModel('itemStyle') as Model<LegendItemStyleOption>;
     const itemStyle = legendItemModel.getItemStyle();
     const iconBrushType = iconType.lastIndexOf('empty', 0) === 0 ? 'fill' : 'stroke';
+    const decalStyle = legendItemModel.getShallow('decal', false);
```

Comment:
Also could you help rename these confusing variables:

`legendModel` should be `legendItemModel`
`legendItemModel` should be `itemStyleModel`

Reply:
> So here we should not passing false to second parameter.

`false` here means not to ignore parent, `decal` is in `legend.itemStyle` so we should pass in `false` to allow it to search in parent model.

>Also could you help rename these confusing variables:

Sure can do.

Reply:
> false here means not to ignore parent, decal is in legend.itemStyle so we should pass in false to allow it to search in parent model.

Sorry my mistake. It's an optional parameter and by default it will search in parent model. So my brain thought you want to stop it when saw the second parameter is used. But still this `false` can be removed because it's the default value.

Reply:
> But still this `false` can be removed because it's the default value.

Hmmm, I am not sure but I think passing `false` explicitly can help debugging in the future. I respect your decision though. Will remove it.

Reply:
> Hmmm, I am not sure but I think passing false explicitly can help debugging in the future

Default means it's the most common used value(here it's more than 90% of the code). Usually we should respect the default and only override it when we know what we are doing in special cases.

Reply:
I see, thank you for your explanation and suggestions.